Homebrew/Resolute Spellbook/firstdaybreak-additions.json

```diff
@@ -0,0 +1,366 @@
+[
+    {
+        "name": "Lightning Surge",
+        "level": 2,
+        "school": "Force",
+        "automation": [
+            {
+                "type": "roll",
+                "roll": "2d6 [lightning]",
+                "name": "damage",
+                "higher": {
+                    "3": "1d6 [lightning]",
+                    "4": "2d6 [lightning]",
+                    "5": "3d6 [lightning]",
+                    "6": "4d6 [lightning]",
+                    "7": "5d6 [lightning]",
+                    "8": "6d6 [lightning]",
+                    "9": "7d6 [lightning]"
+                }
+            },
+            {
+                "type": "target",
+                "target": "self",
+                "effects": [
+                    {
+                        "type": "ieffect2",
+                        "name": "Residual Electricity",
+                        "duration": 1,
+                        "attacks": [
+                            {
+                                "attack": {
+                                    "_v": 2,
+                                    "name": "Residual Electricity - Attack",
+                                    "automation": [
+                                        {
+                                            "type": "target",
+                                            "target": "all",
+                                            "effects": [
+                                                {
+                                                    "type": "ieffect2",
+                                                    "name": "Residual Electricity - Effect",
+                                                    "duration": 1,
+                                                    "desc": "Can't make attacks, no reactions"
+                                                },
+                                                {
+                                                    "type": "damage",
+                                                    "damage": "{ieffect.desc.split()[-1]} [lightning]"
+                                                }
+                                            ]
+                                        }
+                                    ],
+                                    "activation_type": 2
+                                }
+                            }
+                        ],
+                        "buttons": [
+                            {

```

Comment:
This won't work. I would recommend actually making a variable up top called numDie that just returns the number of die you need and then the damage for the primary automation would be {numDie}d6, pass that into the effect description and then the automation would be {ieffect.desc.split()[-1]}d6